Dockerfile

```diff
@@ -4,69 +4,47 @@
 # https://github.com/crate/docker-crate
 #
 
-FROM alpine:3.7
+FROM centos:7
 MAINTAINER Crate.IO GmbH office@crate.io
 
-ENV GOSU_VERSION 1.9
-RUN set -x \
-    && apk add --no-cache --virtual .gosu-deps \
-        dpkg \
-        gnupg \
-        curl \
-    && export ARCH=$(echo $(dpkg --print-architecture) | cut -d"-" -f3) \
-    && curl -o /usr/local/bin/gosu -fSL "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$ARCH" \
-    && curl -o /usr/local/bin/gosu.asc -fSL "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$ARCH.asc" \
-    && export GNUPGHOME="$(mktemp -d)" \
-    && gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
-    && gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu \
-    && rm -rf "$GNUPGHOME" /usr/local/bin/gosu.asc \
-    && chmod +x /usr/local/bin/gosu \
-    && gosu nobody true \
-    && apk del .gosu-deps
+LABEL org.label-schema.schema-version="1.0" \
+    org.label-schema.build-date="2018-11-16T16:10:50.992476076+00:00" \
+    org.label-schema.name="crate" \
+    org.label-schema.description="CrateDB is a distributed SQL database handles massive amounts of machine data in real-time." \
+    org.label-schema.url="https://crate.io/products/cratedb/" \
+    org.label-schema.vcs-url="https://github.com/crate/docker-crate" \
+    org.label-schema.vendor="Crate.io" \
+    org.label-schema.version="3.1.2"
```

Comment:
I'd use the env variable $CRATE_VERSION instead of duplicating the actual version string.

Reply:
I'm not sure you can use env vars in the labels? I'll check though.

Reply:
Regarding the docs it's possible. But having a proper templating mechanism in place will do the job as well.